swagger.json

```diff
@@ -172,6 +172,135 @@
         }
       }
     },
+    "/user/lendinghistory": {
+      "get": {
+        "tags": [
+          "User"
+        ],
+        "summary": "get lending history",
+        "description": "user can see their lending history",
+        "operationId": "get lending history",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful"
+          },
+          "404": {
+            "description": "no lending history",
+            "schema": {
+              "$ref": "#/definitions/error"
+            }
+          },
+          "500": {
+            "description": "Internal server error",
+            "schema": {
+              "$ref": "#/definitions/error"
+            }
+          }
+        },
+        "security": [{
+          "Authorization": []
+        }]
+      }
+    },
+    "/admin/user": {
```

Comment:
You currently have two **/admin/user** in your swagger.json

Reply:
Corrected